source/runtime.cpp

```diff
@@ -839,8 +841,12 @@ void reshade::runtime::load_config()
 			LOG(ERROR) << "Failed to create effect cache directory " << _effect_cache_path << " with error code " << ec.value() << '!';
 	}
 
+	// Use startup preset instead of last selection
+	if (std::error_code ec; !_startup_preset_path.empty() && resolve_preset_path(_startup_preset_path, ec))
```

Comment:
Can re-use the `std:error_code` that is declared in line 833 above here, instead of having to create a new instance.

Reply:
I didn't notice that variable. It will be fixed.